src/index.ts

```diff
@@ -0,0 +1,41 @@
+import {Boom} from '@hapi/boom';
+import {Request, Response} from 'express';
+
+export class BoomableError extends Error {
+  public statusCode = 500;
+}
+
+type ExpressErrorMiddleware = (
+  err: Boom | BoomableError | Error,
+  req: Request,
+  res: Response,
+  next: (err: Error) => void,
+) => void;
+
+export default function (): ExpressErrorMiddleware {
+  return (
+    err: Boom | BoomableError | Error,
+    _req: Request,
+    res: Response,
+    _next: (err: Error) => void,
+  ) => {
+    if (!(err instanceof Error)) {
+      err = new Error(err);
+    }
+    let boom: Boom;
+    if (err instanceof Boom) {
```

Comment:
Use `isBoom` here instead: https://hapi.dev/module/boom/api?v=9.1.4. Unfortunately apps may have multiple versions of boom installed and this ensures that this returns true regardless. `instanceof` will only pass for instances of a unique... instance (lol) of that class.